packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/interfaces.ts

```diff
@@ -29,7 +29,6 @@ export type ConfirmSignupCodeContainerProps = {
 export type ConfirmSignupCodeProps = {
   email: string;
   sessionToken: hexstring;
-  uid: hexstring;
```

Comment:
Heheh, I had removed this in my PR as well until I realized [we need it](https://github.com/mozilla/fxa/pull/16170/files#diff-37b20086028a7347d84b49144135721a746fde9d9b4415b47adb5f20379af569R176).

Reply:
Do we really though? Why not get it from the integration since we're still passing it to the presentation component?

Reply:
It's not always available as a query parameter.